tsl/src/continuous_aggs/create.c

```diff
@@ -722,6 +726,37 @@ caggtimebucket_validate(CAggTimebucketInfo *tbinfo, List *groupClause, List *tar
 						 errmsg(
 							 "time bucket function must reference a hypertable dimension column")));
 
+			if (list_length(fe->args) == 3)
+			{
+				/*
+				 * Currently the third argument of the bucketing function can be
+				 * only a timezone. Only immutable expressions can be specified.
+				 */
+				tz_arg = eval_const_expressions(NULL, lthird(fe->args));
+				if (IsA(tz_arg, Const))
```

Comment:
Just some code style nit pick. If you did:
```
if (!IsA(tz_arg, Const))
	ereport(ERROR,
		(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
		 errmsg("only immutable expressions allowed in time bucket function"),
		 errhint("Use an immutable expression as third argument"
			 " to the time bucket function.")));
```
then you save one level of identation.

Reply:
Good point. I will fix it, thanks